app/models/manageiq/providers/base_manager/refresh_helper_methods.rb

```diff
@@ -0,0 +1,19 @@
+module ManageIQ::Providers::BaseManager::RefreshHelperMethods
+  extend ActiveSupport::Concern
+
+  include Vmdb::Logging
+
+  def memory_logger(ems, message)
+    log_header = format_ems_for_logging(ems)
+    _log.debug("#{log_header} -- #{message} -- Memory usage: #{'%.02f' % collector_memory_usage} MiB")
+  end
+
+  def collector_memory_usage
+    require 'miq-process'
+    MiqProcess.processInfo[:proportional_set_size].to_f / 1.megabyte
```

Comment:
You might want to use `unique_set_size` even though you'll hopefully never need to worry about why it's more helpful for detecting a process growing without bound.  As long as you're looking at the change in these numbers USS should be just as good without the problems detailed in the PR below:

Reference: https://github.com/ManageIQ/manageiq-gems-pending/pull/317